src/gui/spaces/spacesdelegate.cpp

```diff
@@ -169,7 +166,8 @@ QStyleOptionButton SpacesDelegate::openBrowserButtonRect(const QStyleOptionViewI
     opt.iconSize = QSize { px, px };
 
     opt.rect.setSize(QApplication::style()->sizeFromContents(
-        QStyle::CT_PushButton, &opt, opt.fontMetrics.size(Qt::TextSingleLine, opt.text)));
+        QStyle::CT_PushButton, &opt, opt.fontMetrics.size(Qt::TextSingleLine, opt.text), option.widget));
+    opt.rect.setSize(opt.rect.size() + QSize { px * 2, 0 });
```

Comment:
This is just a hack and can't be right.
There needs to be a better solution.

Reply:
Why not leave a TODO in the code?

Reply:
I hoped that we could find a solution in the reviews.

Reply:
```suggestion
    opt.rect.setSize(opt.rect.size() + QSize { px + QApplication::style()->pixelMetric(QStyle::PM_ButtonMargin), 0 });
```

Works pretty well apparently, the button looks like other push buttons with icons:

![screenshot_2022-10-04_16-52-49](https://user-images.githubusercontent.com/80399010/193858125-80410ce5-f35c-4857-bc05-8a546bb5e2a5.png)